### DISCUSSION TITLE
EGSnrc multiple scattering look-up table

### DISCUSSION THREAD

**Discussion by KlaasWillems** [nrc-cnrc/EGSnrc#987] [Questions]
Hi, 

I've been going through the EGSnrc documentation[1]. For my work, I need to sample the multiple scattering distribution based on the screened Rutherford cross-section. The algorithm for this is given in section 2.4.8.i of [1] and involves working with the look-table newms.data, which I found [here](https://github.com/nrc-cnrc/EGSnrc/blob/master/HEN_HOUSE/data/msnew.data) under a slightly different name. I have several questions regarding this look-up table.

1. From my understanding of the [init_ms_SR routine](https://github.com/nrc-cnrc/EGSnrc/blob/master/HEN_HOUSE/src/egsnrc.mortran), the newms.data file contains four lookup tables: ums_array, fms_array, wms_array and ims_array. Which of these contains the multiple scattering distribution q_SR^{2+}? Having looked at the code, I assume it is ums_array, however, could someone confirm?
2. ums_array is three-dimensional (64x8x32). Could you confirm the first dimension corresponds to the ln(\lambda) and the second to G_{1, SR} (notation from section 2.4.8.i in [1])? Having plotted the data, I assume the third dimension is the q_{SR}^{2+} distribution given as a cumulative probability distribution. Is this correct?
3. For which values of ln(\lambda) and G_{1, SR} was the look-up table generated? 

Kind regards, 
Klaas

[1] The EGSnrc Code System: Monte Carlo Simulation of Electron and Photon Transport - Kawrakow et al.

**Comment by rtownson** [comment]
Maybe this is something @eric11210 looked at somewhat recently?

**Reply by eric11210** [reply]
Sorry @rtownson, for some reason I didn't note that I was mentioned here.

@KlaasWillems, unfortunately I haven't into this at all, so I don't have any answers for you anyway.
